src/helpers/overviewReportHelper.js

```diff
@@ -74,6 +74,9 @@ const overviewReportHelper = function () {
           _id: 1,
           firstName: 1,
           lastName: 1,
+          email: 1,
+          profilePic: 1,
+          createdDate:1,
```

Comment:
I don't see field this being used in frontend PR, please kindly correctly if I am wrong.

Reply:
right! not in this case. thanks